src/jdk.crypto.cryptoki/share/native/libj2pkcs11/p11_sessmgmt.c

```diff
@@ -351,7 +378,7 @@ JNIEXPORT void JNICALL Java_sun_security_pkcs11_wrapper_PKCS11_C_1SetOperationSt
 
     free(ckpState);
 
-    if (ckAssertReturnValueOK(env, rv) != CK_ASSERT_OK) { return; }
+    ckAssertReturnValueOK(env, rv);
```

Comment:
Are you going to apply this change to other places? I see 2 new functions still using the old style.

Reply:
Sure, I can change the other 2 places.